activejob/lib/active_job/queue_adapters.rb

```diff
@@ -12,6 +12,8 @@ module ActiveJob
   # * {Sidekiq}[http://sidekiq.org]
   # * {Sneakers}[https://github.com/jondot/sneakers]
   # * {Sucker Punch}[https://github.com/brandonhilkert/sucker_punch]
+  # * {Active Job Async Job}[http://api.rubyonrails.org/classes/ActiveJob/AsyncJob.html]
```

Comment:
You should rather link to the `AsyncAdapter` adapter rather than the `AsyncJob` here, no ?